### DISCUSSION TITLE
Import GLB

### DISCUSSION THREAD

**Discussion by yuchenz27** [atteneder/glTFast#356] [Q&A] [answered]
Can I just drag a GLB file into unity to import?

I have tried this GLB model but it didn't work.

https://cdn.chibilabs.dev/apes/GLBs/1911.glb

I am using Unity2021.2.12 on a Mac :)

Thank you very much.

**Comment by yuchenz27** [comment, marked as answer]
It worked after I installed this package
https://github.com/atteneder/DracoUnity

**Reply by atteneder** [reply]
Excellent! Glad you figured out the solution.
